hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/repository/NftRepositoryTest.java

```diff
@@ -40,4 +40,16 @@ void findById() {
                 .returns(nft.getAccountId(), Nft::getAccountId)
                 .returns(nft.getMetadata(), Nft::getMetadata);
     }
+
+    @Test
+    void count() {
```

Comment:
Should match repo method.
```suggestion
    void countByAccountId() {
```